llvm/include/llvm/CodeGen/StackMaps.h

```diff
@@ -204,6 +204,12 @@ class StackMaps {
         : Reg(Reg), DwarfRegNum(DwarfRegNum), Size(Size) {}
   };
 
+  // Stores callee-saved register info.
+  struct CSR {
+    unsigned Reg = 0;
+    int64_t Offset = 0;
```

Comment:
I think this can be `int` since [getFrameIndex()](https://llvm.org/doxygen/classllvm_1_1CalleeSavedInfo.html#a1876fa422affc81b276851a67859942c) returns `int`?